material-kit-pro-react-v1.9.0/src/index.js

```diff
@@ -30,14 +30,15 @@ import ContactUsPage from "views/ContactUsPage/ContactUsPage.js";
 import EcommercePage from "views/EcommercePage/EcommercePage.js";
 import LandingPage from "views/LandingPage/LandingPage.js";
 import LoginPage from "views/LoginPage/LoginPage.js";
-import PresentationPage from "views/PresentationPage/PresentationPage.js";
+// import PresentationPage from "views/PresentationPage/PresentationPage.js";
```

Comment:
@HassenH1 same here.  I don't want the index.js page for the 3rd party component modified.  create our own index.js.